dashboard-widgets.el

```diff
@@ -710,6 +710,16 @@ String -> bool.
 Argument IMAGE-PATH path to the image."
   (eq 'gif (image-type image-path)))
 
+
+(defun dashboard--is-image-animated-p (image-path)
+  "Return if image has multiple frames.
+String -> bool.
+Argument IMAGE-PATH path to the image."
+   (when (or (eq (image-type image-path) 'gif)
+           (eq (image-type image-path) 'webp))
+   (let ((image (create-image image-path)))
```

Comment:
@ricardoricho @jcs090218 I have now made changes so that the function only checks image type(gif or webp) and multi-frame is checked before animating on this [line](https://github.com/SATYADAHAL/emacs-dashboard/blob/93a70def782a98a47d1d39fea990f9dec8043bf0/dashboard-widgets.el#L771).